src/cross_section/src/cross_section.cpp

```diff
@@ -300,6 +297,7 @@ CrossSection CrossSection::Offset(double delta, JoinType jointype,
 }
 
 double CrossSection::Area() { return C2::Area(paths_); }
+bool CrossSection::IsEmpty() { return paths_.empty(); }
```

Comment:
If paths contains several empty path contours, but no actual points, does this return true or false?

Reply:
Ah it would be false in that case. I'm unsure whether that is a possible case (that clipper would return such a thing from union), but I could add a check for that in anyway.

Reply:
Better yet, add a test - then you'll know if you need to handle the case or not.

Reply:
```
TEST(CrossSection, Empty) {
  Polygons polys(2);
  auto e = CrossSection(polys);
  EXPECT_TRUE(e.IsEmpty());
}
```
Was just coming back to confirm that this passes, so we don't have to worry about it.